containers/hydra/populate.sh

```diff
@@ -11,7 +11,8 @@
 cat /setup/packages.lst |
```

Comment:
Unnecessary use of cat.
```
while ....
    ....
done < /setup/packages.lst
```
Not blocking for this though, unnecessary cats are not included in our bash style guide.